netbox/templates/extras/script_list.html

```diff
@@ -34,7 +34,7 @@ <h5 class="card-header">
                 {% for class_name, script in module_scripts.items %}
                   <tr>
                     <td>
-                      <a href="{% url 'extras:script' module=script.module name=class_name %}" name="script.{{ class_name }}">{{ script }}</a>
+                      <a href="{% url 'extras:script' module=script.module name=class_name %}" name="script.{{ class_name }}">{{ script.Meta.name }}</a>
```

Comment:
`script.Meta.name` is not a required attribute. Instead, this should reference `script.name`.